src/vs/workbench/api/electron-browser/mainThreadWebview.ts

```diff
@@ -200,9 +200,9 @@ export class MainThreadWebviews extends Disposable implements MainThreadWebviews
 			return;
 		}
 
-		const targetGroup = this._editorGroupService.getGroup(viewColumnToEditorGroup(this._editorGroupService, showOptions.viewColumn));
+		const targetGroup = this._editorGroupService.getGroup(viewColumnToEditorGroup(this._editorGroupService, showOptions.viewColumn)) || this._editorGroupService.getGroup(webview.group || 0);
 		if (targetGroup) {
```

Comment:
`targetGroup` is undefined when revealing with `viewColumn.active | undefined` so the `if` block won't be executed.